### ISSUE TITLE
Failure removing files from import

### ISSUE THREAD

**Issue by hallyn** [anuvu/stacker#102]
With the following stacker.yaml:

```
base:
  from:
    type: docker
    url: docker://ubuntu:latest
  run: |
    mkdir -p /boot/a/b
    chmod 444 /boot
test:
  from:
    type: docker
    url: docker://ubuntu:latest
  import:
    - stacker://base/boot/a
  run: |
    echo "hello, world" > /hw
```

doing
```
sudo stacker unpriv-setup
stacker build
```

fails with
```
importing files...
error: couldn't stat import /home/serge/delme/stackertest/roots/base/rootfs/boot/a: lstat /home/serge/delme/stackertest/roots/base/rootfs/boot/a: permission denied
```

While my user owns directory 'boot', it does not have w perms to it, and the kernel's permission checking for directory writes will allow bypassing that restriction only if I actually have CAP_DAC_OVERRIDE against my own user namespace (and the file uid mapped into it), not if i just own the directory.

**Comment by hallyn** [comment]
My plan is at os.Removall(), call a wrapper which checks whether you are in fact uid 0 or have CAP_DAC_OVERRIDE, and if not then use lxc-usernsexec to do the remove.

**Comment by hallyn** [comment]
Oops ignore the specific error above - i used the wrong perms in my demo :)

**Comment by tych0** [comment]
I believe this should be fixed now since #111 was fixed. Please re-open if not.
